lib/xnvme_libconf_entries.c

```diff
@@ -1,11 +1,8 @@
-// SPDX-FileCopyrightText: Samsung Electronics Co., Ltd
-//
-// SPDX-License-Identifier: BSD-3-Clause
-
 ///< This file is autogenerated by 'toolbox/xnvme_libconf.py'
```

Comment:
Since the file is autogenerated, I propose that we remove it from version control

Reply:
To achieve that, additional work is required to create the file when needed, which is beyond the scope of this PR.